.gitignore

```diff
@@ -1,6 +1,5 @@
 node_modules/
 _site/
-_algolia_api_key
```

Comment:
As far as I can tell, this file never existed. See https://github.com/eslint/eslint.github.io/pull/152#issuecomment-155046398 for more details, but here's the relevant bit:

> If you want to push data from your local machine, you just have to use ALGOLIA_API_KEY=xxx jekyll algolia push. On Travis, you just have to declare this variable as an env variable in your config. You can also create a file _algolia_api_key that contains the key, at the root of your jekyll repo (I've ignored it in the .gitignore, just to be safe)